cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -66,13 +66,13 @@ describe("User Creation", () => {
   });
 
   it("Update the existing user profile and verify its reflection", () => {
-    userCreationPage.clickElementById("user-profile-name");
-    userCreationPage.clickElementById("profile-button");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
     userCreationPage.verifyElementContainsText(
       "username-profile-details",
       "devdistrictadmin",
     );
-    userCreationPage.clickElementById("edit-cancel-profile-button");
+    userCreationPage.clickEditCancelProfileButton();
     userCreationPage.typeIntoElementByIdPostClear(
```

Comment:
remove all usage of typeintoelementbyidpostclear all those function with proper individual POM approached function